packages/shorebird_cli/lib/src/shorebird_flutter.dart

```diff
@@ -133,33 +133,40 @@ class ShorebirdFlutter {
   /// Converts a full git revision to a short revision string.
   String shortRevisionString(String revision) => revision.substring(0, 10);
 
+  /// Given a revision and a version, formats them into a single string.
+  ///
+  /// e.g. 3.16.3 and b9b2390296b9b2390296 -> 3.16.3 (b9b2390296)
+  String formatVersion({required String revision, required String? version}) {
+    version ??= 'unknown';
+    return '$version (${shortRevisionString(revision)})';
+  }
+
   /// Returns the current Shorebird Flutter version and revision.
   /// Returns unknown if the version check fails.
   Future<String> getVersionAndRevision() async {
-    String? version = 'unknown';
+    late final String? version;
+
     try {
       version = await getVersionString();
-    } catch (_) {}
+    } on ProcessException {
+      version = 'unknown';
+    }
```

Comment:
why did we change this behavior? Previously we would return `unknown` for any exception and now we're only doing that for a `ProcessException`. Might want to just revert this 🤷

Reply:
Oh good catch! I guess I got confused by the test, which explicit used a `ProcessException`. I will revert here and improve the test too